hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl.tpl

```diff
@@ -61,3 +61,8 @@ assert -name FSMParasiticState::AlertSenderFSMParasiticState_A\
  {!(dut.gen_alert_tx[0].u_prim_alert_sender.state_q inside\
   {Idle, AlertHsPhase1, AlertHsPhase2, PingHsPhase1, PingHsPhase2, Pause0, Pause1}) ->\
   dut.gen_alert_tx[0].u_prim_alert_sender.state_d == Idle}
+
+# These two blocking assignment appears as undetectable and making an assertion for them looks
```

Comment:
Sorry for the nitty comment, but I think you want "These two blocking assignments appear as ..."